cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -576,6 +576,48 @@ def test_self_reported_ethnicity_ontology_term_id__unknown(self, validator_with_
         assert validator.validate_adata()
         assert validator.errors == []
 
+    def test_cell_type_ontology_term_id__unknown(self, validator_with_adata):
+        """
+        Test 'unknown' cell_type_ontology_term_id is valid
+        """
+        validator = validator_with_adata
+        obs = validator.adata.obs
+        obs.at["Y", "cell_type_ontology_term_id"] = "unknown"
+        assert validator.validate_adata()
+        assert validator.errors == []
```

Comment:
i think this line is redundant since `validator.validate_adata()` should only return True if there's no errors

Reply:
Fair, but prefer to leave it explicit esp since this is the pattern used by other tests